bundles/org.openhab.core/src/main/java/org/openhab/core/library/types/ArithmeticGroupFunction.java

```diff
@@ -212,6 +213,82 @@ public State calculate(@Nullable Set<Item> items) {
         }
     }
 
+    /**
+     * This does a logical 'xor' operation. Only if items are of opposing 'activeState' and
+     * 'passiveState', 'activeState' is returned. For more than 2 states, this is '1 of n'.
```

Comment:
> Only if items are of opposing 'activeState' and 'passiveState', 'activeState' is returned

I do not fully get that statement. Could you try to reformulate it to make it clearer?

Reply:
I hope comment is easier to understand now